client/src/app/shared/notification.service.ts

```diff
@@ -0,0 +1,32 @@
+import { Injectable } from '@angular/core';
+
+import * as signalR from '@microsoft/signalr';
+
+import { EnvService } from './env.service';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class NotificationService {
+  private hubConnection: signalR.HubConnection;
+  private readonly apiUrl: string;
+
+  constructor(envService: EnvService) {
+    this.apiUrl = envService.env.apiDomain;
+  }
+
+  startConnection() {
+    this.hubConnection = new signalR.HubConnectionBuilder()
+      .withUrl(`${this.apiUrl}/hub/notifications`)
+      .build();
+
+    this.hubConnection
+      .start()
+      .then(() => console.log('Connection started'))
+      .catch((err) => console.log('Error while starting connection: ' + err));
+
+    this.hubConnection.on('received', ({ source, message }) => {
+      console.log(source, JSON.parse(message));
```

Comment:
Not currently doing anything with the SignalR messages - but this would be pretty easy to change.

Reply:
logging them for now is fine, i brought in ngx-toastr so we can start toasting when we receive messages